AutoMl/tests/System/V1/AutoMlSmokeTest.php

```diff
@@ -77,6 +77,10 @@ public function testAutoMl(AutoMlClient $automl)
             ])
         ]);
 
+        // Note: This line is required. Otherwise test fails citing:
+        // Class google.cloud.automl.v1.OperationMetadata hasn't been added to descriptor pool
+        \GPBMetadata\Google\Cloud\Automl\V1\Operations::initOnce();
```

Comment:
This is concerning because it definitely should not be required for our customers to call this. It should be called internally when `OperationMetadata` is created, or any other message type that exists in that descriptor pool.

So I would say this is a deeper bug, that will require a more comprehensive solution.

Reply:
Another thing interesting is that this does not fail for gRPC, only for the REST transport.

Reply:
Thanks for the review @bshaffer . I raised an internal fix.